.pipelines/templates/template-run-mirror.yml

```diff
@@ -0,0 +1,8 @@
+parameters:
```

Comment:
model this file on template-deploy-azure-env.yml.  You should not need the location, configDirectory or configFileName parameters of it though.  You will need to define additional parameters for dstAuth, dstACRName, etc.